composer.json

```diff
@@ -42,7 +42,7 @@
         "drupal/matomo": "^1.19",
         "drupal/pdf": "^1.1",
         "drupal/rest_oai_pmh": "^2.0@beta",
-        "drupal/search_api_solr": "^4.2",
+        "drupal/search_api_solr": "*",
```

Comment:
This looks like some trickery leftover from fixing the dependency knot? It probably should go back to ^4.2.